packages/client/src/ActionsAndReducers/playerUi/playerUi_Reducer.js

```diff
@@ -242,12 +242,14 @@ export const playerUiReducer = (state = initialState, action) => {
             } = getParticipantStates(channel, newState)
 
             if (allRolesIncluded || isParticipant || newState.isObserver) {
-              const otherParticipants = channel.participants.filter((participant) => participant.forceUniqid !== newState.selectedForce)
               channels[channel.uniqid] = {
                 name: channel.name,
                 templates,
-                forceIcons: otherParticipants.map((participant) => participant.icon),
-                forceColors: otherParticipants.map((participant) => participant.color),
```

Comment:
@zukilover - my implementation was broken, since `participant` doesn't include a `color` parameter, duh.

My solution works, but does the JS look ok to you?

Reply:
I'm still curious why the color prop doesn't exist in `participant` since I'm pretty sure I've added them at:
https://github.com/serge-web/serge/pull/499/files/820385ccf8871d903117bd1cef7ed6e03c227bbc#diff-689bbd4649c073bd63587aa102cafbb3R107 , and
https://github.com/serge-web/serge/pull/499/files/820385ccf8871d903117bd1cef7ed6e03c227bbc#diff-e06bedb132032f61b28920ede710f55eR178

Reply:
No, I don't understand either. But, here is the contents of `channel.participants`:
![image](https://user-images.githubusercontent.com/1108513/95442295-1984c800-0953-11eb-8ac9-128b49bc7000.png)